src/utils/ApiSchema.json

```diff
@@ -59,13 +58,13 @@
       "required": ["upgrades", "vials"]
     },
     "cards": {
-      "description": "Cards tracks the tier of each card (0: unobtained, 1: bronze, 2: silver, 3: gold).",
+      "description": "Cards tracks the tier of each card (0: unobtained, 1: obtained, 2: bronze, 3: silver, 4: gold).",
       "type": "object",
       "patternProperties": {
         "": {
           "type": "integer",
           "minimum": 0,
-          "maximum": 3
+          "maximum": 4
```

Comment:
It would be great to have detailed patterns like this for the other fields, since I had to go poking around in your code to figure out what format the data should be in. E.g. "[A-Z]-\d" regex for constellations, or checking that every statue is `string: {"golden": bool, "level": int, "progress": int}`. I guess your import code will do this implicitly, but it would be nice to have a validator with detailed errors for testing.

Reply:
Apparently `statues` was always well-validated and I forgot about it, so that was a bad example!

Reply:
Sure, I will try to put some regex hints and maybe include some examples in the description of the fields :)

Reply:
Thanks for the hints!